src/components/AddTodo/AddTodo.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import OptionsList from '../OptionsList/OptionsList';
+
+class AddTodo extends Component {
+  state = {
+    changedUser: '0',
+    cardTitle: '',
+    errTitle: '',
+    errChangeUser: '',
+  };
+
+  handleNameChange = (event) => {
```

Comment:
```suggestion
  handleNameChange = ({ target }) => {
```

если мы не используем `event`, почему бы не сделать деструктуризацию?